iot_packet_verifier/src/verifier.rs

```diff
@@ -475,15 +479,17 @@ mod test {
         assert_eq!(
             valid_packets,
             vec![
+                // TODO: some weird shit with the timestamp values in the tests
```

Comment:
i've never had luck testing things where timing is variable without setting a pre-determined timestamp at the beginning of the operation